database/src/migrations/release.0.26/biohub.sql

```diff
@@ -1,3575 +1,3746 @@
+;
+COMMENT ON COLUMN survey_proprietor.revision_count IS 'Revision count used for concurrency control.'
+;
+COMMENT ON TABLE survey_proprietor IS 'Intersection table associating surveys to proprietary types and associated meta data.'
+;
+
+-- 
+-- TABLE: survey_spatial_component 
+--
+
+CREATE TABLE survey_spatial_component(
+    survey_spatial_component_id    integer                     GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    survey_id                      integer                     NOT NULL,
+    name                           varchar(50)                 NOT NULL,
+    description                    varchar(3000),
+    geometry                       geometry(geometry, 3005),
+    geography                      geography(geometry),
+    create_date                    timestamptz(6)              DEFAULT now() NOT NULL,
+    create_user                    integer                     NOT NULL,
+    update_date                    timestamptz(6),
+    update_user                    integer,
+    revision_count                 integer                     DEFAULT 0 NOT NULL,
+    CONSTRAINT "PK200" PRIMARY KEY (survey_spatial_component_id)
+)
+;
+
+
+
+COMMENT ON COLUMN survey_spatial_component.survey_spatial_component_id IS 'System generated surrogate primary key identifier.'
+;
+COMMENT ON COLUMN survey_spatial_component.survey_id IS 'System generated surrogate primary key identifier.'
+;
+COMMENT ON COLUMN survey_spatial_component.name IS 'The name of the record.'
+;
+COMMENT ON COLUMN survey_spatial_component.description IS 'The description of the record.'
+;
+COMMENT ON COLUMN survey_spatial_component.geometry IS 'The containing geometry of the record.'
+;
+COMMENT ON COLUMN survey_spatial_component.geography IS 'The containing geography of the record.'
+;
+COMMENT ON COLUMN survey_spatial_component.create_date IS 'The datetime the record was created.'
+;
+COMMENT ON COLUMN survey_spatial_component.create_user IS 'The id of the user who created the record as identified in the system user table.'
+;
+COMMENT ON COLUMN survey_spatial_component.update_date IS 'The datetime the record was updated.'
+;
+COMMENT ON COLUMN survey_spatial_component.update_user IS 'The id of the user who updated the record as identified in the system user table.'
+;
+COMMENT ON COLUMN survey_spatial_component.revision_count IS 'Revision count used for concurrency control.'
+;
+COMMENT ON TABLE survey_spatial_component IS 'Survey spatial component persists the various spatial components that a survey may include. Examples are block and transect spatial artifacts.'
+;
+
+-- 
+-- TABLE: survey_summary_detail 
+--
+
+CREATE TABLE survey_summary_detail(
+    survey_summary_detail_id       integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    id                             integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
```

Comment:
Is this `id` intentional? Or a duplicate of `survey_summary_detail_id`?

Reply:
some wierd-ness going on... will fix.